src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
```

Comment:
The 'hidden' property should not be part of the Animal class constructor as not all animals will have this property. It's specific to Herbivores, so it should only be defined in the Herbivore class.